src/pcb/pcb_missing_footprint_error.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from "zod"
+import { getZodPrefixedIdWithDefault } from "src/common"
+import { expectTypesMatch } from "src/utils/expect-types-match"
+
+export const pcb_missing_footprint_error = z
+  .object({
+    type: z.literal("pcb_missing_footprint_error"),
+    pcb_missing_footprint_error_id: getZodPrefixedIdWithDefault(
+      "pcb_missing_footprint_error",
+    ),
+    error_type: z.literal("pcb_missing_footprint_error"),
+    message: z.string(),
```

Comment:
Can you include an optional source component id

Reply:
Sure, I've added the filed 👍🏻